index.js

```diff
@@ -10,7 +10,14 @@ const androidWebkitVersions = require('./constants/androidWebkitVersions')
 
 const ANDROID_FLAVOURS = {
   KITKAT: '4.4.0',
-  MARSHMELLOW: '6.0.0',
+  LOLLIPOP: '5.0',
+  MARSHMALLOW: '6.0.0',
+  NOUGAT: '7.0',
+  OREO: '8.0',
+  PIE: '9.0',
+  ANDROID_10: 'Android 10',
```

Comment:
`Android` prefix is already present below —
https://github.com/pastelsky/useragent-generator/pull/4/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R110

Reply:
I didn't understand - the 2 groups have different meaning, correct?

Reply:
These need to be semver numbers, otherwise the comparisons below are not going to work.

Reply:
Done.